packages/rancher-monitoring/prometheus-federator/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,11 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -6,7 +6,7 @@
+   catalog.cattle.io/os: linux,windows
+   catalog.cattle.io/permits-os: linux,windows
+   catalog.cattle.io/provides-gvr: helm.cattle.io.projecthelmchart/v1alpha1
+-  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <=2.6.99-0'
++  catalog.cattle.io/rancher-version: '>= 2.6.5-0 <=2.6.99-0'
```

Comment:
Can you make this `'>= 2.6.5-0 <= 2.6.100-0'`?

Reply:
Sure